application/controllers/UserManagementController.php

```diff
@@ -1150,21 +1213,57 @@ public function actionTakeOwnership()
      */
     public function deleteUser(int $uid): bool
```

Comment:
Why is the actionDeleteUser function not using this function?
It's doing basically the same isn't it?

Reply:
Because someone thought it would be funny to have 2 different systems :).
The real answer would be that the massive action renders a completely different modal which expects other values and formats.

Reply:
nice ;)